consonance-webservice/src/main/java/io/swagger/wes/api/Ga4ghApi.java

```diff
@@ -0,0 +1,127 @@
+package io.swagger.wes.api;
```

Comment:
Yup, we went down this road with Dockstore. It is entirely too tempting to place hacks in the generated code then another developer will be unaware and blow it away with a new run of generation. We're experimenting with simply not checking in the generated code on the server-side* with some creative use of the [parameters](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen-maven-plugin/README.md#general-configuration-parameters) to avoid generating certain bits, override package names, and override the occasional template. 

* we've used this approach on the client-side for a long time with success, but the same strategy on the server side is more difficult thus more recent

Will tell you two how it goes.